microlith/loki/alerting/couchbase/couchbase-rules.yaml

```diff
@@ -181,3 +181,23 @@ groups:
           title: Analytics Crash ({{ $labels.hostname }})
           description: "Analytics on node: {{ $labels.hostname }} has crashed."
           remediation: Review analytics_debug.log to identify the cause, or contact Couchbase Technical Support.
+
+      - alert: CB90052-malformedvBuckets
+        expr: |
+          count_over_time({file="memcached.log"} |~ ".*malformed data in file.*" |json message="message" | line_format "{{.message}}" | pattern "<_>WARNING (<bucket>)<_>malformed data in file<_>, <vbucket>" [1h]) > 0
```

Comment:
`|~ ".*malformed data in file.*"` can just be `|= "malformed data in file"`, and it'll be a little cheaper since we're not using regexes